common/djangoapps/third_party_auth/provider.py

```diff
@@ -16,18 +17,19 @@ class Registry(object):
     @classmethod
     def _enabled_providers(cls):
         """ Helper method to iterate over all providers """
+        site = get_current_site()
         for backend_name in _PSA_OAUTH2_BACKENDS:
-            provider = OAuth2ProviderConfig.current(backend_name)
+            provider = OAuth2ProviderConfig.current(get_current_site(), backend_name)
```

Comment:
@bdero nit: `get_current_site` is already saved in `site` variable two lines above.